src/xmipp/tests/test_binding.py

```diff
@@ -120,8 +86,31 @@ def test_xmipp_Euler_matrix2angles(self):
         self.assertAlmostEqual(tilt, tilt1, 4)
         self.assertAlmostEqual(psi, psi1, 4)
 
+        # test that slicing the array won't affect the result
+        rot = tilt = psi = 0.0
+        A = arange(16.0).reshape(4, -1)
+        B = A[:3,:3]
+        rot1,tilt1,psi1 = Euler_matrix2angles(B)
+        self.assertAlmostEqual(48.36646, rot1, 4)
```

Comment:
This test fails here

`AssertionError: 48.36646 != 49.39870535499554 within 4 places (1.0322453549955455 difference)
`

Reply:
maybe related to https://github.com/I2PC/xmipp/pull/686/files

Reply:
Yes, these tests assume that it has been already fixed. Sorry for not being clear about that.
Can I ask you to run them again?

Reply:
Ok all the binding tests Pass